sycl/unittests/scheduler/EnqueueWithDependsOnDeps.cpp

```diff
@@ -332,6 +332,9 @@ TEST_F(DependsOnTests, ShortcutFunctionWithWaitList) {
   ASSERT_NE(Cmd, nullptr);
   Cmd->MIsBlockable = true;
   Cmd->MEnqueueStatus = detail::EnqueueResultT::SyclEnqueueBlocked;
+  // for the test functionality, depenent task HostTaskEvent must be treated as
```

Comment:
I would replace this comment with "Mock up an incomplete host task" on L326.

Reply:
Done.